### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by viccuad**
Update links and explanation on the readme. Tried to make it as concise as possible.
I'm open to suggestions.

### CODE REVIEW

**Comment by viccuad** [comment]
I would like to update the motif on the repo, from  "SCF on top of Kind" to "CI for KubeCF, SCF & Stratos", or "Deployment tool for KubeCF, SCF & Stratos", or something similar :sweat_smile: .

File: README.md [hunk 1]
```diff
@@ -1,15 +1,54 @@
-# [![Build Status](https://travis-ci.com/SUSE/catapult.svg?branch=master)](https://travis-ci.com/SUSE/catapult) catapult: [SCF](https://github.com/SUSE/scf) on top of [Kind](https://github.com/kubernetes-sigs/kind) (and others..)
+# [![Build  Status](https://travis-ci.com/SUSE/catapult.svg?branch=master)](https://travis-ci.com/SUSE/catapult) CI implementation for [KubeCF](https://github.com/SUSE/kubecf), [SCF](https://github.com/SUSE/scf) & [Stratos](https://github.com/cloudfoundry/stratos)
```
Comment (mudler) [inline README.md]: I would leave at this point just "Catapult" in the headers no? and we could add `CI implementation for [KubeCF](https://github.com/SUSE/kubecf), [SCF](https://github.com/SUSE/scf) & [Stratos](https://github.com/cloudfoundry/stratos)` as a short info right after , wdyt?

File: README.md [hunk 2]
```diff
@@ -1,15 +1,54 @@
...
+
+# Description
+
+Catapult is a CI implementation that is from the ground-up designed to work
+locally. This allows iterating and using it for manual tests and development of
+the product, in addition to running it in your favourite CI scheduler
+(Concourse, Gitlab…).
+
+Catapult supports several k8s backends: can create CaaSP4, GKE, EKS clusters on its
+own, and you can bring your own cluster with the "imported" backend.
```
Comment (mudler) [inline README.md]: I would add a reference in the wiki about this feature

Comment (viccuad) [reply README.md]: Instead of in the readme, or in addition to the readme?

File: README.md (hunk 1 above)
Comment (viccuad) [reply README.md]: true! done!

**Comment by viccuad** [comment]
thanks for the review! merged because I got one colleague asking about how to run it, so better have it there.

File: README.md (hunk 2 above)
Comment (viccuad) [reply README.md]: Updated https://github.com/SUSE/catapult/wiki/Imported

**Pull request merged**